src/data/contributors/Hex-24.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Hex-24",
+  "skills": ["C++", "Python", "ML"],
+  "cause": "Self Growth:)",
+  "image": "Avatar4",
+  "github": "https://github.com/hex-24",
+  "twitter": "https://twitter.com/Hex-24,
```

Comment:
@ashis-solomon just remove the comma from the last last json object and your code will run fine. I guess there is a bug in gatsby json parser.